pasqal_cloud/utils/constants.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+
+class JobOrigin(Enum):
```

Comment:
There is no real reason for a user to filter on this so I don't think we need to expose it in the SDK. All batches created with the SDK should have the "CLIENT" origin